.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
me parece que estos archivos son específicos para IntelliJ’s, no seria bueno incluirlos en el proyecto

https://rider-support.jetbrains.com/hc/en-us/articles/207097529-What-is-the-idea-folder-